src/app/pages/i18n/i18n.component.html

```diff
@@ -0,0 +1,91 @@
+                    <mat-icon matListIcon>home</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HOME' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleI18nMenu()">
+                    <mat-icon matListIcon>folder</mat-icon>
+                    <span>{{ 'MENU_ITEMS.PROJECTS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleI18nMenu()">
+                    <mat-icon matListIcon>report</mat-icon>
+                    <span>{{ 'MENU_ITEMS.WARNINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleI18nMenu()">
+                    <mat-icon matListIcon>settings</mat-icon>
+                    <span>{{ 'MENU_ITEMS.SETTINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleI18nMenu()">
+                    <mat-icon matListIcon>help</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HELP' | translate }}</span>
+                </a>
+            </div>
+        </mat-nav-list>
+    </mat-sidenav>
+    <mat-sidenav-content class="content">
+        <mat-toolbar color="primary">
+            <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+                <mat-icon>menu</mat-icon>
+            </button>
+            <div>{{ 'I18N' | translate }}</div>
+            <pxb-spacer></pxb-spacer>
+            <button
+                [matTooltip]="'I18N_MENU_TOOLTIP' | translate"
+                matTooltipPosition="below"
+                matTooltipHideDelay="200"
+                type="button"
+                aria-label="Toggle sidenav"
+                mat-icon-button
+                (click)="toggleI18nMenu()"
+                style="margin-left: -4px; margin-right: 12px;"
+            >
+                <mat-icon [class.rtl]="rtl" aria-label="Side nav toggle icon">menu_open</mat-icon>
+            </button>
+        </mat-toolbar>
+
+        <mat-toolbar class="select-language">
+            <mat-form-field>
+                <mat-select
+                    #langeSelect
+                    (ngModelChange)="translate.use(langeSelect.value)"
+                    [(ngModel)]="selectedLanguage"
+                >
+                    <mat-option *ngFor="let lang of enabledLocales; let i = index" [value]="enabledLocales[i]">
+                        {{ 'LANGUAGES.' + lang | translate }}
+                    </mat-option>
+                </mat-select>
+            </mat-form-field>
+            <button mat-raised-button color="primary" style="margin: 20px;">
+                <mat-icon [class.rtl]="rtl">shopping_cart</mat-icon> {{ 'ADD_TO_CART' | translate }}
+            </button>
+        </mat-toolbar>
+
+        <mat-nav-list>
+            <pxb-info-list-item
+                *ngFor="let fruit of fruits"
+                (click)="toggleFruit(fruit)"
+                [chevron]="true"
+                [hidePadding]="true"
```

Comment:
maybe false for now so it looks a little more like a typical ILI

Reply:
Yep, this'll make things easier :)